avocado/utils/network/interfaces.py

```diff
@@ -313,6 +318,36 @@ def save(self, ipaddr, netmask):
                       'IPV6_DEFROUTE': 'yes'}
         if current_distro.name == 'SuSE':
             ifcfg_dict.pop('BOOTPROTO')
+
+        if self.if_type == 'Bond':
+            # Update master and slave cfg for bond
+            ifcfg_dict.update({'BONDING_MASTER': 'yes'})
+            mode = self._get_bondinterface_details().splitlines()[0].split()[0]
+            slaves = self._get_bondinterface_details().splitlines()[1].split()
```

Comment:
You are calling the same command twice. One suggestion is to change the method so it will return a tuple.

```python
mode, slaves = self._get_bondinterface_details()
```